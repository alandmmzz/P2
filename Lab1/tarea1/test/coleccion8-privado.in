crearColeccion

crearFecha 01/01/1987
crearLibro 361177880 "Beloved" "Toni" "Morrison" "Una esclava liberada es perseguida por el fantasma de su hija muerta" 1
agregarEnColeccion

crearFecha 01/01/1999
crearLibro 381199002 "Desgracia" "J.M." "Coetzee" "Un profesor sudafricano cae en desgracia y enfrenta una realidad post-apartheid" 1
agregarEnColeccion

crearFecha 01/01/2002
crearLibro 351166779 "Kafka en la orilla" "Haruki" "Murakami" "Dos historias paralelas entrelazadas por lo onírico, la soledad y la identidad" 1
agregarEnColeccion

crearFecha 01/01/1951
crearLibro 341155668 "El hombre ilustrado" "Ray" "Bradbury" "Cuentos de ciencia ficción unidos por los tatuajes animados de un misterioso hombre" 5
agregarEnColeccion

crearFecha 11/11/1954
crearLibro 331144557 "Las dos torres" "J.R.R." "Tolkien" "La comunidad del anillo se divide y continúa su lucha contra el poder oscuro" 6
agregarEnColeccion

crearFecha 01/01/1983
crearLibro 301112345 "La pianista" "Elfriede" "Jelinek" "Una profesora de piano vive una vida reprimida bajo el dominio de su madre" 1
agregarEnColeccion

crearFecha 01/01/1989
crearLibro 391200113 "Los restos del día" "Kazuo" "Ishiguro" "Un mayordomo rememora su vida y su lealtad ciega durante entreguerras" 1
agregarEnColeccion

crearFecha 01/08/1965
crearLibro 161718192 "Dune" "Frank" "Herbert" "Una lucha política y espiritual en el desierto del planeta Arrakis" 5
agregarEnColeccion

crearFecha 01/09/1977
crearLibro 311122335 "El resplandor" "Stephen" "King" "Una familia se aísla en un hotel donde fuerzas sobrenaturales influyen en la mente del padre" 2
agregarEnColeccion

crearFecha 02/07/1998
crearLibro 321133446 "Harry Potter y la cámara secreta" "J.K." "Rowling" "Harry regresa a Hogwarts y enfrenta el misterio de una cámara oculta" 6
agregarEnColeccion

crearFecha 01/01/2003
crearLibro 371188991 "La noche del oráculo" "Paul" "Auster" "Un escritor se obsesiona con una historia que parece volverse real" 1
agregarEnColeccion

crearFecha 01/01/1990
crearLibro 401211224 "Una cuestión personal" "Kenzaburō" "Ōe" "Un joven enfrenta el dilema moral de tener un hijo con daño cerebral" 1
agregarEnColeccion

imprimirColeccion

removerDeColeccion 3
removerDeColeccion 7
removerDeColeccion 10

imprimirColeccion

imprimirLibrosGeneroColeccion 1

imprimirLibrosGeneroColeccion 17

liberarColeccion

Fin