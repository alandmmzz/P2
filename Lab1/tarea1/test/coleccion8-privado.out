1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>33>34>35>36>37>38>Libro 341155668
Título: El hombre ilustrado
Autor: Ray Bradbury
Descripción: Cuentos de ciencia ficción unidos por los tatuajes animados de un misterioso hombre
Género: 5
Fecha de edición: 1/1/1951
Libro 331144557
Título: Las dos torres
Autor: J.R.R. Tolkien
Descripción: La comunidad del anillo se divide y continúa su lucha contra el poder oscuro
Género: 6
Fecha de edición: 11/11/1954
Libro 161718192
Título: Dune
Autor: Frank Herbert
Descripción: Una lucha política y espiritual en el desierto del planeta Arrakis
Género: 5
Fecha de edición: 1/8/1965
Libro 311122335
Título: El resplandor
Autor: Stephen King
Descripción: Una familia se aísla en un hotel donde fuerzas sobrenaturales influyen en la mente del padre
Género: 2
Fecha de edición: 1/9/1977
Libro 301112345
Título: La pianista
Autor: Elfriede Jelinek
Descripción: Una profesora de piano vive una vida reprimida bajo el dominio de su madre
Género: 1
Fecha de edición: 1/1/1983
Libro 361177880
Título: Beloved
Autor: Toni Morrison
Descripción: Una esclava liberada es perseguida por el fantasma de su hija muerta
Género: 1
Fecha de edición: 1/1/1987
Libro 391200113
Título: Los restos del día
Autor: Kazuo Ishiguro
Descripción: Un mayordomo rememora su vida y su lealtad ciega durante entreguerras
Género: 1
Fecha de edición: 1/1/1989
Libro 401211224
Título: Una cuestión personal
Autor: Kenzaburō Ōe
Descripción: Un joven enfrenta el dilema moral de tener un hijo con daño cerebral
Género: 1
Fecha de edición: 1/1/1990
Libro 321133446
Título: Harry Potter y la cámara secreta
Autor: J.K. Rowling
Descripción: Harry regresa a Hogwarts y enfrenta el misterio de una cámara oculta
Género: 6
Fecha de edición: 2/7/1998
Libro 381199002
Título: Desgracia
Autor: J.M. Coetzee
Descripción: Un profesor sudafricano cae en desgracia y enfrenta una realidad post-apartheid
Género: 1
Fecha de edición: 1/1/1999
Libro 351166779
Título: Kafka en la orilla
Autor: Haruki Murakami
Descripción: Dos historias paralelas entrelazadas por lo onírico, la soledad y la identidad
Género: 1
Fecha de edición: 1/1/2002
Libro 371188991
Título: La noche del oráculo
Autor: Paul Auster
Descripción: Un escritor se obsesiona con una historia que parece volverse real
Género: 1
Fecha de edición: 1/1/2003
39>No se puede remover el libro con isbn 3 porque NO pertenece a la colección.
40>No se puede remover el libro con isbn 7 porque NO pertenece a la colección.
41>No se puede remover el libro con isbn 10 porque NO pertenece a la colección.
42>Libro 341155668
Título: El hombre ilustrado
Autor: Ray Bradbury
Descripción: Cuentos de ciencia ficción unidos por los tatuajes animados de un misterioso hombre
Género: 5
Fecha de edición: 1/1/1951
Libro 331144557
Título: Las dos torres
Autor: J.R.R. Tolkien
Descripción: La comunidad del anillo se divide y continúa su lucha contra el poder oscuro
Género: 6
Fecha de edición: 11/11/1954
Libro 161718192
Título: Dune
Autor: Frank Herbert
Descripción: Una lucha política y espiritual en el desierto del planeta Arrakis
Género: 5
Fecha de edición: 1/8/1965
Libro 311122335
Título: El resplandor
Autor: Stephen King
Descripción: Una familia se aísla en un hotel donde fuerzas sobrenaturales influyen en la mente del padre
Género: 2
Fecha de edición: 1/9/1977
Libro 301112345
Título: La pianista
Autor: Elfriede Jelinek
Descripción: Una profesora de piano vive una vida reprimida bajo el dominio de su madre
Género: 1
Fecha de edición: 1/1/1983
Libro 361177880
Título: Beloved
Autor: Toni Morrison
Descripción: Una esclava liberada es perseguida por el fantasma de su hija muerta
Género: 1
Fecha de edición: 1/1/1987
Libro 391200113
Título: Los restos del día
Autor: Kazuo Ishiguro
Descripción: Un mayordomo rememora su vida y su lealtad ciega durante entreguerras
Género: 1
Fecha de edición: 1/1/1989
Libro 401211224
Título: Una cuestión personal
Autor: Kenzaburō Ōe
Descripción: Un joven enfrenta el dilema moral de tener un hijo con daño cerebral
Género: 1
Fecha de edición: 1/1/1990
Libro 321133446
Título: Harry Potter y la cámara secreta
Autor: J.K. Rowling
Descripción: Harry regresa a Hogwarts y enfrenta el misterio de una cámara oculta
Género: 6
Fecha de edición: 2/7/1998
Libro 381199002
Título: Desgracia
Autor: J.M. Coetzee
Descripción: Un profesor sudafricano cae en desgracia y enfrenta una realidad post-apartheid
Género: 1
Fecha de edición: 1/1/1999
Libro 351166779
Título: Kafka en la orilla
Autor: Haruki Murakami
Descripción: Dos historias paralelas entrelazadas por lo onírico, la soledad y la identidad
Género: 1
Fecha de edición: 1/1/2002
Libro 371188991
Título: La noche del oráculo
Autor: Paul Auster
Descripción: Un escritor se obsesiona con una historia que parece volverse real
Género: 1
Fecha de edición: 1/1/2003
43>Libro 301112345
Título: La pianista
Autor: Elfriede Jelinek
Descripción: Una profesora de piano vive una vida reprimida bajo el dominio de su madre
Género: 1
Fecha de edición: 1/1/1983
Libro 361177880
Título: Beloved
Autor: Toni Morrison
Descripción: Una esclava liberada es perseguida por el fantasma de su hija muerta
Género: 1
Fecha de edición: 1/1/1987
Libro 391200113
Título: Los restos del día
Autor: Kazuo Ishiguro
Descripción: Un mayordomo rememora su vida y su lealtad ciega durante entreguerras
Género: 1
Fecha de edición: 1/1/1989
Libro 401211224
Título: Una cuestión personal
Autor: Kenzaburō Ōe
Descripción: Un joven enfrenta el dilema moral de tener un hijo con daño cerebral
Género: 1
Fecha de edición: 1/1/1990
Libro 381199002
Título: Desgracia
Autor: J.M. Coetzee
Descripción: Un profesor sudafricano cae en desgracia y enfrenta una realidad post-apartheid
Género: 1
Fecha de edición: 1/1/1999
Libro 351166779
Título: Kafka en la orilla
Autor: Haruki Murakami
Descripción: Dos historias paralelas entrelazadas por lo onírico, la soledad y la identidad
Género: 1
Fecha de edición: 1/1/2002
Libro 371188991
Título: La noche del oráculo
Autor: Paul Auster
Descripción: Un escritor se obsesiona con una historia que parece volverse real
Género: 1
Fecha de edición: 1/1/2003
44>45>46>Fin.
